src/manual/index.md

```diff
@@ -2,14 +2,45 @@
 title: TaskCluster Manual
 ---
 
-TaskCluster is a collection of microservices that together form a distributed
-task-execution framework supporting Mozilla's continuous integration.
+What does TaskCluster do?
 
-Most of these microservices provide REST APIs, and [accessing those
-APIs](/manual/apis) is an important part of interacting with TaskCluster.
-Several TaskCluster components also [watch version-control
-repositories](/manual/vcs), so it is also possible to interact with TaskCluster
-without ever making an API call.
+Fundamentally, it executes *tasks*. A task is defined in a JSON object, and
+placed on a *queue*. A *worker* later claims that task, executes it, and
+updates the task with the results.
 
-In any case, TaskCluster is focused on [tasks](/manual/tasks) and on [executing
-those tasks](/manual/execution) efficiently.
+## The TaskCluster Service and Team
+
+We provide TaskCluster as a service, rather than as an installable application.
```

Comment:
Was this section copy-paste? If so, I understand the double spaces. Could you replace the double spaces in this section with single spaces?